docs/courses/robotics/3.3-liquid-handlers.md

```diff
@@ -7,11 +7,59 @@
 
 ## 🔰 Tutorial
 
-In this module, you will perform liquid transfer between containers with an automated liquid handler (Science Jubilee and Opentrons OT-2).
+Welcome to the fascinating world of automated liquid handling! 🌊 In this module, you'll learn how to perform liquid transfers between containers using two cutting-edge systems: the Science Jubilee and the Opentrons OT-2. 
+
+### The Liquid Handling Superstars 🌟
+
+1. **Science Jubilee** 🔬
+   Imagine a Swiss Army knife for scientists - that's Science Jubilee! It's a versatile, open-source platform that can handle various lab tasks, including precise liquid transfers.
+
+   Fun fact: Science Jubilee is like a scientific chameleon, adapting to different research needs! 🦎
```

Comment:
Let's clarify that it's an open source toolchanger at heart. You can refer to the science jubilee docs for a definition (and we should probably link to these docs anyway https://science-jubilee.readthedocs.io/)

Reply:
EDIT: I'm also seeing the more detailed section below

Reply:
Maybe for the fun fact, you can point to the research article where it was used for quantum dot synthesis: https://pubs.rsc.org/en/content/articlelanding/2023/dd/d3dd00033h